Controller/SuggestionProviderController.php

```diff
@@ -86,4 +103,38 @@ public function addSuggestionProvider(SuggestionProviderInterface $matcher, $gro
     {
         $this->suggestionProviders[] = array('provider' => $matcher, 'group' => $group);
     }
+
+    /**
+     * You can define your templates for each format in the bundle's configuration.
+     *
+     * @param string $format
+     *
+     * @return string
+     */
+    private function getTemplateForSuggestions($format = 'html')
+    {
+        if (!isset($this->configuration['templates'][$format])) {
+            throw new InvalidConfigurationException(
+                sprintf(
+                    'No template for format %s defined. Use one of %s or configure one for your format.',
+                    $format,
+                    implode(', ', array_keys($this->configuration['templates']))
+                )
+            );
+        }
+
+        return $this->configuration['templates'][$format];
+    }
+
+    /**
+     * @return bool
+     */
+    private function isExcluded(Request $request)
+    {
+        foreach ($this->configuration['excluded_paths'] as $path) {
```

Comment:
you could use the request matcher for this. that is a class in symfony that can check a request on path but also controller name and whatnot.